### ISSUE TITLE
Support channel creation + publishing in same block

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbryum-server#53]
A common scenario that occurs within the LBRY app and on spee.ch is where a user creates a channel and then tries to publish content to it shortly after. Per @jackrobison this is an issue on lbryum-server being able to support the publish tx while the channel creation is waiting for a confirmation in the same block. 

This used to be possible at some point, and then we ran into https://github.com/lbryio/lbry/issues/1037. Now that the hard error is fixed, the exception just tells you that you can't publish to the channel - doesn't seem informative enough for the situation at hand as the same error happens if you try to publish to an invalid channel.

**Comment by kauffj** [comment]
So this did used to work? What changed to make it not possible? Is it expected to be made possible again? @jackrobison @lyoshenka 

This is a very common error and source of confusion app side, so we need to decide whether/when it'll get fixed or the app team needs to paper over it. @liamcardenas

**Comment by tzarebczan** [comment]
@shyba we should test this scenario out on lbryum-x when you are ready.

**Comment by shyba** [comment]
https://github.com/lbryio/lbry/issues/1037 was closed with a fix on the client side, while channel creation and publish on the same block is possible using `channel_id` (spee.ch is doing this way). 

Using `channel_id` for same block channel+publish works on lbryum-server and lbryumx as far as I tested. However, using `channel_name` triggers an electrum query on the client side and raises an error because the server doesn't return unconfirmed claims. This is expected and maybe a documentation note should be added to the daemon API parameter `channel_name` on publish call.
